node/full_client.go

```diff
@@ -496,7 +496,11 @@ func (c *FullClient) Commit(ctx context.Context, height *int64) (*ctypes.ResultC
 	if err != nil {
 		return nil, err
 	}
-	commit := com.ToABCICommit(heightValue, b.Hash())
+
+	// we only have one centralize sequencer = 1 validators
+	val := b.SignedHeader.Validators.Validators[0].Address
```

Comment:
[nit] we should verify `Validators` isn't empty before grabbing an element from the slice
```suggestion
        if len(b.SignedHeader.Validators.Validators[0]) == 0 {
            return nil, errors.New("empty validator set found in block")
        }
	val := b.SignedHeader.Validators.Validators[0].Address
```